tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/pullquote_block.html

```diff
@@ -1,26 +1,51 @@
-{% load wagtailcore_tags %}
-<blockquote>
-    <svg width="120" height="90" aria-hidden="true">
-        <use xlink:href="#quote-icon" />
+{% load wagtailcore_tags wagtailimages_tags %}
+<blockquote class="blockquote">
+
+    {# This needs to be inline as masks or clipPaths are not supported in <use> elements #}
+    <svg class="blockquote__icon" width="75" height="60" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+        <mask id="quote-with-motif-mask" maskUnits="userSpaceOnUse" x="25" y="-4" width="71" height="60">
```

Comment:
If this quote is used more than once on a page then this will be flagged as a duplicate ID. Is there a variable we can append here to make it unique?

Reply:
yes there is probably a block id. I will see if I can find what that would be.

Reply:
Here's how to do it (there may be a better way...)

Edit this line of code https://github.com/torchbox/torchbox.com/blob/integration/sprint-1/tbx/project_styleguide/templates/patterns/molecules/streamfield/stream_block.html#L6 to be `{% include_block block with unique_id=block.id %}`

And then in the quote template you can use `{{ unique_id }}` - you'll have to also pass that through to the svg include when you move it.

Reply:
This is perfect, thanks. Added in https://github.com/torchbox/torchbox.com/pull/26/commits/5f809dd58c906df27ab67a741a5150506fc818e2 and working as expected!